src/Microsoft.ML.FastTree/FastTreeStatic.cs

```diff
@@ -105,6 +105,48 @@ public static class FastTreeStatic
             return rec.Output;
         }
 
+        /// <summary>
+        /// FastTree <see cref="RankerContext"/>.
+        /// </summary>
+        /// <param name="ctx">The <see cref="RegressionContext"/>.</param>
+        /// <param name="label">The label column.</param>
+        /// <param name="features">The features colum.</param>
+        /// <param name="groupId">The name of the groupId column.</param>
```

Comment:
>The name of the groupId column [](start = 34, length = 30)

It's not really a *name* is it? The point of pigsty is you don't use names, you just pass in objects.